_posts/2015-01-08-creating-a-federal-marketplace-for-agile-delivery-services.md

```diff
@@ -45,7 +45,7 @@ Any good economist will tell you that competition is key to a healthy marketplac
 To encourage healthy competition, we’re going to do two things: use simple and transparent contractor performance metrics and encourage federal buyers to structure their acquisitions to use many small, short-duration contracts (referred to as [modular contracting](https://www.whitehouse.gov/sites/default/files/omb/procurement/guidance/modular-approaches-for-information-technology.pdf)). By doing so, we’ll quickly gain visibility into who is and who isn’t delivering value to the government. Those who are will earn greater opportunities to win more work. Those who aren’t will be periodically off-ramped and competitively replaced with new marketplace entrants.
 
 ##Small business friendly
-Small businesses are the drivers of economic growth in the U.S. Agile methodologies encourage work to be done in small, rapid, independent increments. When combined with modular approaches to contracting, this opens up a whole new world of contract opportunities for small businesses. For example, rather than compete work as one big, long contract (for example, $50 million over five years) that typically only large businesses can execute on, break it into many small, short-duration contracts (for example, ten $5 million contracts of three to six months in duration). This has the effect of leveling the competitive playing field for not just small companies, but businesses of all sizes. Federal buyers will be strongly encouraged to structure their acquisitions this way. For those who need help, the services of [18F Consulting](https://18f.gsa.gov/consulting/) will be made available to them.
+Small businesses are the drivers of economic growth in the U.S. Agile methodologies encourage work to be done in small, rapid, independent increments. When combined with modular approaches to contracting, this opens up a whole new world of contract opportunities for small businesses. For example, rather than compete work as one big, long contract (for example, $50 million over five years) that typically only large businesses can execute on, break it into many small, short-duration contracts (for example, ten $5 million contracts of three to six months in duration). This has the effect of leveling the competitive playing field for not just small companies, but businesses of all sizes. Federal buyers will be strongly encouraged to structure their acquisitions this way. For those who need help, the services of [18F Consulting](/consulting/) will be made available to them.
```

Comment:
We will need to add `{{ site.baseurl }}` at the beginning of each root relative link

For instance:

```
[18F Consulting]({{ site.baseurl }}/consulting/)
```